scripts/parse-arguments.sh

```diff
@@ -111,6 +112,10 @@ for argument in "$@"; do
     EXTERNAL_LIBRARIES+=("libx264")
     FFMPEG_GPL_ENABLED=true
     ;;
+  --enable-libbluray | -libbluray)
+    EXTERNAL_LIBRARIES+=("libfreetype")
```

Comment:
I hope in some future I'll create a proper transitive dependencies for the supported libraries.
For now, please avoid enabling 2 libraries like this. Users may enable both libbluray and libfreetype by flags, which will result in building the libfreetype twice.
Let's stick to the necessity of manual adding both libbluray and libfreetype if you need to build just the libbluray.

Reply:
ok